### PR TITLE
fix(p-search-dropdown): add `selected-extra` slot

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console(if usages are changed) 

### Description
SSIA
스크린샷 2022-12-13 오후 2 34 29

### CODE REVIEW

**Review by minseolee — APPROVED** [review]
LGTM!!

**Review by piggggggggy — APPROVED** [review]
LGTM!!!!!

File: src/inputs/dropdown/search-dropdown/story-helper.ts [hunk 1]
```diff
@@ -197,6 +198,15 @@ const extraArgTypes: ArgTypes = {
             type: { summary: null },
         },
     },
+    'selected-extra': {
```
Comment (WANZARGEN) [inline src/inputs/dropdown/search-dropdown/story-helper.ts]: nitpick: This blocks duplication of prop name and slot name. What about changing it to 'selectedExtraSlot'?

**Review by WANZARGEN — APPROVED** [review]
LGTM~ I left a nitpick comment :)

**Pull request merged**